Source/ChocolateyGui.Common/Properties/Resources.resx

```diff
@@ -1241,6 +1241,9 @@ Please contact your System Administrator to enable this operation.</value>
   <data name="SettingsView_TogglePreventUsageOfUpdateAllButtonDescription" xml:space="preserve">
     <value>Prevents the ability for user to use the Update All button. NOTE: This feature will only work with Chocolatey for Business and the Chocolatey GUI licensed extension installed.</value>
   </data>
+  <data name="SettingsView_ToggleAllowGlobalConfirmationDescription" xml:space="preserve">
+    <value>Does something, for some time...</value>
```

Comment:
That is an interesting description of the configuration value 😆

Reply:
I've updated this description to be a little bit more descriptive 😁